code/src/index.css

```diff
@@ -11,3 +12,505 @@ code {
+.footer-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+.progress-wrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.label-progress {
+  display: flex;
+  height: 4vw;
+  font-family: 'Montserrat';
+  margin: 0 10% 0 10%;
+  letter-spacing: 0.1em;
+  font-size: 17px;
+}
+/*Button section. The span creates the arrow, and depending which button you press it moves in a specific direction.*/
+.button-next {
+  display: flex;
+  color: var(--buttonText);
+  background-color: var(--buttonBackground);
+  padding: 10px 16px;
+  border-radius: 20px;
+  transition: all .3s ease;
+  font-weight: bold;
+  cursor: pointer;
+  align-items: center;
+  font-size: 14px;
+}
+
+.button-next > .arrow {
+  width: 6px;
+  height: 6px;
+  border-right: 2px solid var(--buttonArrow);
+  border-bottom: 2px solid var(--buttonArrow);
+  position: relative;
+  transform: rotate(-45deg);
+  margin: 0 6px;
+  transition: all .3s ease;
+}
+
+.button-next > .arrow::before {
+  display: block;
+  background-color: currentColor;
+  width: 3px;
+  transform-origin: bottom right;
+  height: 2px;
+  position: absolute;
+  opacity: 0;
+  bottom: calc(-2px / 2);
+  transform: rotate(45deg);
+  transition: all .3s ease;
+  content: "";
+  right: 0;
+}
+
+.button-next:hover > .arrow {
+  transform: rotate(-45deg) translate(4px, 4px);
+  border-color: text-hover-color;
+}
+
+.button-next:hover > .arrow::before {
+  opacity: 1;
+  width: 8px;
+}
+
+.button-next:hover {
+  background-color: var(--buttonHover);
+  color: var(--buttonHoverText);
+}
+
+.button-previous {
+  display: flex;
+  color: var(--buttonText);
+  background-color: var(--buttonBackground);
+  padding: 10px 16px;
+  border-radius: 20px;
+  transition: all .3s ease;
+  font-weight: bold;
+  cursor: pointer;
+  align-items: center;
+  font-size: 14px;
+}
+
+.button-previous > .arrow {
+  width: 6px;
+  height: 6px;
+  border-right: 2px solid var(--buttonArrow);
+  border-bottom: 2px solid var(--buttonArrow);
+  position: relative;
+  transform: rotate(135deg);
+  margin: 0 6px;
+  transition: all .3s ease;
+}
+
+.button-previous > .arrow::before {
+  display: block;
+  background-color: currentColor;
+  width: 3px;
+  transform-origin: bottom right;
+  height: 2px;
+  position: absolute;
+  opacity: 0;
+  bottom: calc(-2px / 2);
+  transform: rotate(45deg);
+  transition: all .3s ease;
+  content: "";
+  right: 0;
+}
+
+.button-previous:hover > .arrow {
+  transform: rotate(135deg) translate(4px, 4px);
+  border-color: text-hover-color;
+}
+
+.button-previous:hover > .arrow::before {
+  opacity: 1;
+  width: 8px;
+}
+
+.button-previous:hover {
+  background-color: var(--buttonHover);
+  color: var(--buttonHoverText);
+}
+
+.button-submit {
+  display: flex;
+  color: var(--buttonText);
+  background-color: var(--buttonSubmit);
+  padding: 10px 16px;
+  border-radius: 20px;
+  transition: all .3s ease;
+  font-weight: bold;
+  cursor: pointer;
+  align-items: center;
+  font-size: 14px;
+}
```

Comment:
Really like that the submit button is blue! With the styling of the progressbar (that's also blue), you really gets that you've reached the end of the survey and that you then intuitively need to press the submit button.